addons/portal/views/assets.xml

```diff
@@ -11,7 +11,7 @@
         </xpath>
     </template>
 
-    <template id="assets_frontend" inherit_id="web_editor.assets_frontend" name="Portal Assets" priority="15">
+    <template id="assets_frontend" inherit_id="web_editor.assets_frontend" name="Portal Assets">
```

Comment:
It was a fix https://github.com/odoo/odoo/commit/74d72b9153e8e710d081db386975d35458724b89
Probably need a double check